lib/mongoid/enum.rb

```diff
@@ -1,5 +1,6 @@
 require "mongoid/enum/version"
 require "mongoid/enum/validators/multiple_validator"
+require 'active_support/inflector'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.